pkg/versiongetter/github_release.go

```diff
@@ -26,28 +27,31 @@ type GitHubReleaseClient interface {
 	ListReleases(ctx context.Context, owner, repo string, opts *github.ListOptions) ([]*github.RepositoryRelease, *github.Response, error)
 }
 
-func (g *GitHubReleaseVersionGetter) Get(ctx context.Context, pkg *registry.PackageInfo, filters []*Filter) (string, error) {
+func (g *GitHubReleaseVersionGetter) Get(ctx context.Context, logE *logrus.Entry, pkg *registry.PackageInfo, filters []*Filter) (string, error) {
 	repoOwner := pkg.RepoOwner
 	repoName := pkg.RepoName
 
-	release, _, err := g.gh.GetLatestRelease(ctx, repoOwner, repoName)
+	var respToLog *github.Response
+	defer func() {
+		logGHRateLimit(logE, respToLog)
+	}()
```

Comment:
This seems to output the info log about GitHub API rate limit always.
I don't want to output the log always.
This kind of logs should be outputted as debug log, or should be outputted as error log when an API rate limit error occurs.